### ISSUE TITLE
Committees page

### ISSUE THREAD

**Issue by JacalynLaird** [softwaresaved/event-organisation-guide#33]
Hyperlink not working: "details of what committee roles entail"

Can this be reworded? I find it hard to follow:
(e.g. suitable contacts, sponsors who are willing for the steering committee and local contact who can help with logistics for the organising committee)

**Comment by shoaibsufi** [comment]
I might need to update Claire to get the details

**Comment by shoaibsufi** [comment]
eps-committees.rst is updated
